src/nvim/message.c

```diff
@@ -227,7 +227,7 @@ void msg_multiline_attr(const char *s, int attr)
 {
   const char *next_spec = s;
 
-  while (next_spec != NULL) {
+  while (next_spec != NULL && !got_int) {
```

Comment:
Perhaps this is too general. The previous implementation used `got_int` only in `ex_echo`. But this function is also uses for certain error messages (like lua errors), seems there is a risk that such an error message caused by an interrupt can get inhibited.

 `msg_multiline_attr` could take a `bool` flag which is only set to true for the usage in `ex_echo`, to follow the previously correct implementation more closely.

Reply:
ok, done.